src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -125,8 +125,7 @@ public void AddInterceptionListener(IInterceptionListener listener)
             this.interceptionListeners.AddFirst(listener);
         }
 
-        [SuppressMessage("Microsoft.Design", "CA1033:InterfaceMethodsShouldBeCallableByChildTypes",
-            Justification = "Explicit implementation to be able to make the IFakeCallProcessor interface internal.")]
+        [SuppressMessage("Microsoft.Design", "CA1033:InterfaceMethodsShouldBeCallableByChildTypes", Justification = "Explicit implementation to be able to make the IFakeCallProcessor interface internal.")]

```

Comment:
I hesitated between putting everything on one very long line and putting each parameter on its own line... But I guess it's less intrusive this way.